javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,132 @@
+// FLIGHT BOOKING
+
+function getFullname(firstName, surName, useFormalName) {
+    const fullName = `${firstName} ${surName}`;
+    if (firstName === "" || surName === "") {
+      alert("Please provide first and   last names!");
+      return ""; 
+    } else {
+      return useFormalName ? `Lord ${fullName}` : fullName;
+    }
+  }
+  
+  let fullname1 = getFullname("Richard", "Sanchez", false);
+  let fullname2 = getFullname("Sponge", "Bob", true);
+  console.log(fullname1);
+  console.log(fullname2);
+  
+  // EVENT APPLICATION
+  
+  const today = new Date();
+  const todayNumber = today.getDay();
+  
+  function getEventWeekday(daysFromToday) {
+    const eventDays = (todayNumber + daysFromToday) % 7;
+    const week = [
+      "Sunday",
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+    ];
+  
+    return week[eventDays];
+  }
+  
+  const eventDate = getEventWeekday(13);
+  console.log(eventDate);
+  
+  // WEATHER WEAR
+  
+  function whatToWear(temperature) {
+    let youShouldWear;
+    if (temperature <= 5) {
+      youShouldWear = "Long-sleeved fur jumpsuit";
+    } else if (temperature > 5 && temperature <= 13) {
+      youShouldWear = "Long-sleeved wool jumpsuit";
+    } else if (temperature > 13 && temperature <= 22) {
+      youShouldWear = "Long-sleeved cotton jumpsuit and a jacket";
+    } else if (temperature > 22 && temperature <= 30) {
+      youShouldWear = "short-sleeved cotton jumpsuit";
+    } else if (temperature > 30 && temperature <= 35) {
+      youShouldWear = "Shorts and a tank-top";
+    } else {
+      youShouldWear = "Stay in a cool pool!";
+    }
+    return youShouldWear;
+  }
+  
+  const clothesToWear = whatToWear(36);
+  console.log(clothesToWear);
+  
+  // STUDENT MANAGER
+ 
+  
+  let class07Students = [];
+  
+  function addStudentToClass(studentName) {
```

Comment:
This function return only ´cannot add more students´, it does not even add the queen.
Try to refractor the function and check if it outputs all the cases.
You can write this function with only if... if else statements, you don't need nested if statements.